schedule/qam/common/qam-yast_self_update_libyui-rest.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: qam-yast_self_update_libyui-rest
+description: self_update via INSTALLER repo & efi test on UEFI
+vars:
+  YUI_REST_API: 1
+schedule:
+  - installation/bootloader_start
+  - installation/setup_libyui
+  - installation/product_selection/install_SLES
+  - installation/validate_self_update
+  - installation/licensing/accept_license
+  - installation/registration/register_via_scc
+  - installation/module_registration/register_module_desktop
```

Comment:
This part of the [ticket](https://progress.opensuse.org/issues/112574) is missing:
`We will need some adaptions related with the module selection, in Product validation we have [these 4 test cases](https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/tests/installation/module_selection) and there are more product modules to install in MU.`